test/interface.jl

```diff
@@ -0,0 +1,242 @@
+            Normal(0.1, 2.5),
+            Pareto(),
+            Rayleigh(1.0),
+            TDist(2),
+            TruncatedNormal(0, 1, -Inf, 2),
+        ]
+        
+        for dist in uni_dists
+            @testset "$dist: dist" begin
+                td = transformed(dist)
+
+                # single sample
+                y = rand(td)
+                x = inv(td.transform)(y)
+                @test logpdf(td, y) ≈ logpdf_with_trans(dist, x, true)
+
+                # logpdf_with_jac
+                lp, logjac = logpdf_with_jac(td, y)
+                @test lp ≈ logpdf(td, y)
+                @test logjac ≈ logabsdetjacinv(td.transform, y)
+
+                # multi-sample
+                y = rand(td, 10)
+                x = inv(td.transform).(y)
+                @test logpdf.(td, y) ≈ logpdf_with_trans.(dist, x, true)
+
+                # logpdf corresponds to logpdf_with_trans
+                d = dist
+                b = bijector(d)
+                x = rand(d)
+                y = b(x)
+                @test logpdf(d, inv(b)(y)) - logabsdetjacinv(b, y) ≈ logpdf_with_trans(d, x, true)
+                @test logpdf(d, x) + logabsdetjac(b, x) ≈ logpdf_with_trans(d, x, true)
+            end
+
+            @testset "$dist: ForwardDiff AD" begin
+                x = rand(dist)
+                b = DistributionBijector{Bijectors.ADBackend(:forward_diff), typeof(dist)}(dist)
+                
+                @test abs(det(Bijectors.jacobian(b, x))) > 0
+                @test logabsdetjac(b, x) ≠ Inf
+
+                y = b(x)
+                b⁻¹ = inv(b)
+                @test abs(det(Bijectors.jacobian(b⁻¹, y))) > 0
+                @test logabsdetjac(b⁻¹, y) ≠ Inf
+            end
+
+            @testset "$dist: Tracker AD" begin
+                x = rand(dist)
+                b = DistributionBijector{Bijectors.ADBackend(:reverse_diff), typeof(dist)}(dist)
+                
+                @test abs(det(Bijectors.jacobian(b, x))) > 0
+                @test logabsdetjac(b, x) ≠ Inf
+
+                y = b(x)
+                b⁻¹ = inv(b)
+                @test abs(det(Bijectors.jacobian(b⁻¹, y))) > 0
+                @test logabsdetjac(b⁻¹, y) ≠ Inf
+            end
+        end
+    end
+
+    @testset "Truncated" begin
+        d = Truncated(Normal(), -1, 1)
+        b = bijector(d)
+        x = rand(d)
+        @test b(x) == link(d, x)
+
+        d = Truncated(Normal(), -Inf, 1)
+        b = bijector(d)
+        x = rand(d)
+        @test b(x) == link(d, x)
+
+        d = Truncated(Normal(), 1, Inf)
+        b = bijector(d)
+        x = rand(d)
+        @test b(x) == link(d, x)
+    end
+
+    @testset "Multivariate" begin
+        vector_dists = [
+            Dirichlet(2, 3),
+            Dirichlet([1000 * one(Float64), eps(Float64)]),
+            Dirichlet([eps(Float64), 1000 * one(Float64)]),
+            MvNormal(randn(10), exp.(randn(10))),
+            # MvLogNormal(MvNormal(randn(10), exp.(randn(10)))),
```

Comment:
Why do we comment out this one?

Reply:
Numerical inaccuracies using `DistributionBijector` (which used `ADBijector` as the transform). But I've now implemented `bijector(d::LogMvNormal) = log_b`, which of course does not suffer from the same, so it's now fixed:)